allennlp/common/registrable.py

```diff
@@ -3,17 +3,18 @@
 any base class with a named registry for its subclasses and a decorator
 for registering them.
 """
-from collections import defaultdict
-from typing import TypeVar, Type, Callable, Dict, List, Optional, Tuple
+import collections
```

Comment:
Any reason for this change?

Reply:
It made the `_registry: ...` annotation and assignment wrap in a more visually-pleasing way 🙃

The alternative is

```py3
class Registrable(FromParams):
    _registry: ClassVar[DefaultDict[type, Dict[str, Tuple[type, Optional[str]]]]] = defaultdict(
        dict
    )
```

Another option is to break it up onto two lines, if the wrapping bothers you as much as it bothers me.

```py3
class Registrable(FromParams):
    _registry: ClassVar[DefaultDict[type, Dict[str, Tuple[type, Optional[str]]]]]
    _registry = defaultdict(dict)
```

Only downside is that we're repeating `_registry` twice.

And a final option is to break down that rather hefty `_registry` annotation into chunks using type aliases. Arguably, that's probably the "most correct" option but naming things is hard. I'd have to give it some thought as to what to call the pieces.

Let me know what you prefer!

Reply:
Ah, I see. You could also disable auto-formatting for this block by adding the `# fmt: off` comment before and the `# fmt: on` comment after.

Reply:
Pushing a change. Let me know if this is okay.